OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -427,7 +433,9 @@ public override bool Tick(Actor self)
 				if (progress >= Distance)
 				{
 					mobile.SetCenterPosition(self, To);
-					mobile.Facing = ToFacing;
+					mobile.Facing = TurnsWhileMoving
+						? Util.TickFacing(mobile.Facing, FromToYaw, mobile.TurnSpeed)
+						: WAngle.Lerp(FromFacing, ToFacing, progress, Distance);
```

Comment:
```suggestion
						: ToFacing;
```

Reply:
well this is beyond this PR, it belongs to normal move behaivior.

Reply:
no, you are changing it. See the value you are replacing

Reply:
my bad

Reply:
fixed